.travis.yml

```diff
@@ -28,4 +28,4 @@ before_install:
 
 # Identical to package.json except for travis wait
 script:
-  - skyux lint && skyux test && skyux e2e && travis_wait skyux build --logFormat none --output-keep-alive
+  - skyux lint && skyux test && skyux e2e && skyux build
```

Comment:
We had to introduce the `travis_wait` due to long build times.  Is there a reason to remove it now?

Reply:
Ah, this was unintentional (though, removing it doesn't seem to affect the build). I'll put it back.

Reply:
Thanks!